README.md

```diff
@@ -74,6 +74,16 @@ Editing the `preprompts`, and evolving how you write the project prompt, is how
 
 Each step in `steps.py` will have its communication history with GPT4 stored in the logs folder, and can be rerun with `scripts/rerun_edited_message_logs.py`.
 
+### Running with open source models
```

Comment:
This section might go better in the docs (which I just found), although concerned the instructions will simply be buried there. Maybe we could just do it, throw in a link from the README, and call it a day.